astropy/time/formats.py

```diff
@@ -85,26 +85,26 @@ class TimeFormatMeta(type):
 
     _registry = TIME_FORMATS
 
-    def __new__(mcls, name, bases, members):
-        cls = super().__new__(mcls, name, bases, members)
+    def __new__(cls, name, bases, members):
```

Comment:
Use standard idiom `__new__(cls, ...)`.

Reply:
But this actually is a metaclass, no? I'd keep it as is.

Reply:
Done.